src/ray/raylet/local_object_manager.cc

```diff
@@ -73,6 +76,7 @@ void LocalObjectManager::ReleaseFreedObject(const ObjectID &object_id) {
     if (pinned_objects_.count(object_id)) {
       pinned_objects_size_ -= pinned_objects_[object_id]->GetSize();
       pinned_objects_.erase(object_id);
+      pinned_object_owners_.erase(object_id);
```

Comment:
I think we should probably do this outside of the if check, in case the object was spilled.